.github/workflows/rust.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         with:
           command: test
           args: -p tendermint --test integration --no-fail-fast -- --ignored
+      - uses: actions-rs/cargo@v1
```

Comment:
I believe it would be better to isolate the jobs per crate. i.e. `test-integration-light-client-stable` under jobs.

Reply:
Alright!

Reply:
Done

Reply:
Dope!